src/middlewared/middlewared/plugins/system.py

```diff
@@ -356,12 +356,133 @@ def autotune(self, conf='loader'):
 
 class SystemService(Service):
 
+    DMIDECODE_CACHE = {
+        'ecc-memory': None,
+        'baseboard-manufacturer': None,
+        'baseboard-product-name': None,
+        'system-manufacturer': None,
+        'system-product-name': None,
+        'system-serial-number': None,
+        'system-version': None,
+    }
+
+    CPU_INFO = {
+        'cpu_model': None,
+        'core_count': None,
+    }
+
+    MEM_INFO = {
+        'physmem_size': None,
+    }
+
+    BIRTHDAY_DATE = {
+        'birth': None,
```

Comment:
I was basically thinking that do we require this to be a dict ? How about storing it directly ?

Reply:
Oh yeah, makes sense. I'll fix